pallets/anchors-v2/src/benchmarking.rs

```diff
@@ -0,0 +1,89 @@
+// Copyright 2021 Centrifuge Foundation (centrifuge.io).
+// This file is part of Centrifuge chain project.
+
+// Centrifuge is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version (see http://www.gnu.org/licenses).
+
+// Centrifuge is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+use frame_benchmarking::{account, benchmarks, impl_benchmark_test_suite};
+use frame_support::traits::Currency;
+use frame_system::RawOrigin;
+use parity_scale_codec::EncodeLike;
+use scale_info::prelude::format;
+use sp_core::H256;
+use sp_runtime::traits::Hash;
+
+use super::*;
+
+benchmarks! {
+	where_clause {
```

Comment:
There is a `benchmarking::v2` that we should use instead, for reference: `pallet_investments` among others.

Reply:
Yup, thanks for the pointer ❤️